lib/records/I_RecCore.h

```diff
@@ -168,11 +169,15 @@ int RecGetRecordBool(const char *name, RecBool *rec_byte, bool lock = true);
 //------------------------------------------------------------------------
 // Record Attributes Reading
 //------------------------------------------------------------------------
+// Values with names that have this prefix are exempted from being considered "unrecognized".
+const char REC_PLUGIN_CONFIG_NAME_PREFIX[] = "proxy.config.plugin.";
+static const size_t REC_PLUGIN_CONFIG_NAME_PREFIX_LEN = sizeof(REC_PLUGIN_CONFIG_NAME_PREFIX)-1;
```

Comment:
You can't get the value of the record unless it is registered, so I don't think this saves you any work.